flask_resty/testing.py

```diff
@@ -1,12 +1,36 @@
+from datetime import datetime, timedelta
+import re
+
+from dateutil import parser
```

Comment:
right I agree. I think all these suite of assertion helpers (Shape, JUST_BEFORE, Matches) could make up a testing library of its own :D